apps/root/src/services/labelService.ts

```diff
@@ -1,30 +1,36 @@
-import { AccountLabels } from '@types';
+import { AccountLabels, AccountLabelsAndContactList, ILabelService } from '@types';
 import AccountService from './accountService';
 import MeanApiService from './meanApiService';
+import ContactListService from './conctactListService';
 
-export default class LabelService {
+export default class LabelService implements ILabelService {
   labels: AccountLabels = {};
 
   meanApiService: MeanApiService;
 
   accountService: AccountService;
 
-  constructor(meanApiService: MeanApiService, accountService: AccountService) {
+  contactListService: ContactListService;
+
+  constructor(meanApiService: MeanApiService, accountService: AccountService, contactListService: ContactListService) {
     this.meanApiService = meanApiService;
     this.accountService = accountService;
+    this.contactListService = contactListService;
   }
 
   getStoredLabels(): AccountLabels {
     return this.labels;
   }
 
-  async fetchLabels(): Promise<void> {
+  async fetchLabelsAndContactList(): Promise<AccountLabelsAndContactList | undefined> {
     const user = this.accountService.getUser();
     if (!user) {
       return;
     }
+
     try {
-      this.labels = await this.meanApiService.getAccountLabels(user.id);
+      const accountLabelsAndContactList = await this.meanApiService.getAccountLabelsAndContactList(user.id);
```

Comment:
you can just return `this.meanApiService.getAccountLabelsAndContactList(user.id)`

Reply:
Sweet!